lib/web_ui/tool/unicode_sync_script.dart

```diff
@@ -305,17 +304,17 @@ class PropertyCollection {
         .map(parseLineIntoUnicodeRange)
         .toList();
     // Insert the default property if it doesn't exist.
-    final EnumValue found = enumCollection.values.firstWhere(
-      (property) => property.name == defaultProperty,
+    final EnumValue? found = enumCollection.values.cast<EnumValue?>().firstWhere(
```

Comment:
The items in the list are not nullable. Can we remove the nullability from `cast<EnumValue?>`?

Reply:
The items in the original list are not nullable, but `orElse` makes the result nullable. In order to allow that, I first need to convert the list items to nullable so that `orElse` is allowed to return `null`.